src/components/HomepageTextIntro.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import TextIntro from '../components/shared-components/text-intro/TextIntro';
+
+const HomepageIntro = () => {
+  return <TextIntro location='homepage' />;
+};
+
+export default HomepageIntro;
```

Comment:
Can just push the TextIntro straight into the homepage with <TextIntro location='homepage' />

Reply:
Unless this method is required for the page layout to work then no worries